docs/telepresence/2.3/install/migrate-from-legacy.md

```diff
@@ -95,5 +94,5 @@ Flags:
   -n, --namespace string   If present, the namespace scope for this CLI request
 ```
 
-Since the new architecture deploys a traffic-manager into the Ambassador namespace, please take a look at
+Since the new architecture deploys a Traffic Manager into the Ambassador namespace, please take a look at
```

Comment:
this should also be `traffic-manager`

Reply:
Same as on https://github.com/datawire/getambassador.io/pull/1088 let's address that after the release (so these 2 PRs stay in sync).